pkgs/development/libraries/arrayfire-ml/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchurl, fetchFromGitHub, cmake, pkgconfig, arrayfire }:
+
+let
+  version = "0.1.0";
+in stdenv.mkDerivation rec {
+  pname = "arrayfire-ml";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "arrayfire";
+    repo = "arrayfire-ml";
+    rev = "fde97a5bf157936c8d904d0c857c6cb3275ee527";
+    sha256 = "0ij6g36zcxqgcyhcy28ycg7jf1hgw4p9hv5l3jmkzis0yqnxm346";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+
+  buildInputs = [ arrayfire ];
+
+  installPhase = ''
+    mkdir -p $out/lib
+    mkdir -p $out/LICENSES
+    mkdir -p $out/share
+
+    cp libafml.so* $out/lib/.
+    cp -r ${src}/include $out/.
+    cp ${src}/LICENSE $out/LICENSES/BSD\ 3-Clause.txt
+    cp -r ${src}/examples $out/share/.
```

Comment:
you don't need to reference ${src} here, it's automatically extracted to the current working directory